### DISCUSSION TITLE
Version 2.1.0

### DISCUSSION THREAD

**Discussion by shazzaam7** [xenia-manager/xenia-manager#211] [General]
#  Xenia Manager v2.1.0 Release 

## 📝 Notes
- This version is not compatible with v1 of Xenia Manager. It needs a full fresh reinstall. 
- Anyone who already installed v2.0.0 and has issues, it is recommended to do a fresh install of Xenia emulators to ensure that everything works just fine. If it doesn't fix the issue with the configuration file, please do a fresh install (Sorry about that, I was sure that it was working fine because it ended up working on 3 of my PC's)

## 🚀 What's New  
# ✨ General Improvements  
- **Xenia Settings Cleanup:** 
	- Replaced `ge_menu_sensitivity` with `menu_sensitivity`
	- Settings that are missing from the configuration file will be hidden in the UI (To prevent the current situation with `ge_menu_sensitivity`

# 🎮 Feature Enhancements  
- **Improved `Content Viewer`:** Now it shows the actual `gamertags` alongside `xuid` for profiles in the `Content Viewer` 
 
# 🛠️ Bugfix
- **Automatic Save Backup:** Due to a typo it was borked, now it should work.
- **Symbolic Link Backup Method:** Added a backup method in case the creation of SymLinks of the configuration files fails

Happy holidays, and thank you for your continued support! 🎉

<hr /><em>This discussion was created from the release <a href='https://github.com/xenia-manager/xenia-manager/releases/tag/2.1.0'>Version 2.1.0</a>.</em>

**Comment by h34rt-0f-1c3** [comment]
Just joined GitHub. Thank you for your work on the Xenia Manager! Suggestion: a UI button toggle to enable XBLA games (license_mask) & achievement pop-ups would be useful instead of having to edit the Canary config file (which tends to reset itself to default values).

**Reply by shazzaam7** [reply]
Both of them are already implemented as `License Mask` and `Show Achievement Notifications` in the `Xenia Settings` tab.
